docs/content/users/install/ddev-installation.md

```diff
@@ -354,6 +353,15 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
     }
     ```
 
+    The universal image used above already installs `docker-in-docker`. If you use another image which does not include it, you'll need to add it:
```

Comment:
I'm not entirely sure we need to add this section. What do you think?

Reply:
_Side note: I haven't had time to figure out if custom images like `image": "mcr.microsoft.com/devcontainers/go:0-1-bullseye",`  need `docker-in-docker` or if they install it.  There were some background info given here: https://github.com/devcontainers/features/issues/634#issuecomment-1682620353_

For simplicity reasons we could definitely remove this paragraph, especially when the universal image is recommended. 

I'll remove it.

Reply:
Removed it ✅